downstream/modules/aap-on-azure/proc-peer-vwan-hub-to-aap.adoc

```diff
@@ -0,0 +1,33 @@
+[id="proc-peer-vwan-hub-to-aap"]
+
+= Peering a VWAN Hub to the {AAPonAzureNameShort} Network
+
+Before you can peer a VWAN with the {AAPonAzureNameShort} VNET, you must have connected a hub network and at least one spoke network to the hub network of the Azure VWAN that you intend to connect {AAPonAzureNameShort} to.
```

Comment:
It  sounds a little odd to say 'connect to the thing you intend to connect to'. Perhaps 'that you intend to use'? Just a suggestion.

Reply:
Yes: this was a comment from the SME

> @ariordan-redhat Nuts, I jumped the gun again with the peer review. Sorry. I didn't notice the WIP label and just went for it after I got the email notification. I can close all my comments if you prefer.
> 
> Maybe we could remove that code owners thing and create a peer review group? That way, when we're ready for review, we can add the label and then tag the group or individual reviewers.

No problem. I forgot to remove the WIP tag.

Reply:
@ianf77 Would this work?

"Before peering, you must connect a hub network and at least one spoke network to the hub network of the Azure VWAN that you intend to connect {AAPonAzureNameShort} to."

I think the SME wanted to be completely clear that the required network connections needed to be made to the specific WAN that you're connecting AAP to.

Reply:
It's a weird sentence but I can't really think of a  way round it if the 'intend to connnect to' is also necessary